tests.py

```diff
@@ -323,8 +328,32 @@ def test_missing_publickey(self):
         with self.assertRaises(sender.MissingFile):
             ss.send_cpm(0, 0)
 
-    # ...
+  
+class TestDataLog(unittest.TestCase):
+    
+    def setUp(self):
+        print('Checking local data')
+        
+    def test_get_data(self):
+        mgr = Manager(data=True, test=True, interval=10)
+        this_start, this_end = mgr.get_interval(time.time())
+        time.sleep(10)
+        mgr.handle_cpm(this_start, this_end)
+        this_start, this_end = mgr.get_interval(time.time())
+        time.sleep(10)
+        mgr.handle_cpm(this_start, this_end)
```

Comment:
These 6 lines (`this_start, this_end` ... `mgr.handle_cpm`) are basically duplicating what Manager normally does internally, which is not so nice. Also, 20 seconds is a long time to add to tests. Hmm, I realize you can't do Manager.run() because it blocks the program, but we can speed this up. Try:
```
now = time.time()
mgr.handle_cpm(now - 10, now)
[mgr.sensor.count() for _ in xrange(2)]
mgr.handle_cpm(now, now + 10)
```
The `mgr.sensor.count()` throws a couple artificial counts in there (in case that changes any behavior anywhere, which right now it doesn't).
The start time and end time don't have to match the actual execution time, and 10 could be anything here as long as `now + 10` is later than the execution time so that the counts are included. See `Sensor.get_cpm()` if this is confusing.